types/ioredis/index.d.ts

```diff
@@ -632,6 +634,8 @@ declare namespace IORedis {
         smembers(key: KeyType, callback?: (err: Error, res: any) => void): Pipeline;
 
         zadd(key: KeyType, ...args: string[]): Pipeline;
+        zadd(key: KeyType, score: number, member: string): Pipeline;
+        zadd(key: KeyType, ...args: Array<[ number, string ] | string>): Pipeline;
```

Comment:
this signature is not combined, please combine this signature with first overload.